client/blocks/eligibility-warnings/hold-list.tsx

```diff
@@ -128,26 +107,16 @@ function getBlockingMessages( translate: LocalizeProps[ 'translate' ] ) {
 			contactUrl: null,
 		},
 		SITE_GRAYLISTED: {
-			message: hasLocalizedText(
+			message: translate(
 				"There's an ongoing site dispute. Contact us to review your site's standing and resolve the dispute."
-			)
-				? hasLocalizedText(
-						"There's an ongoing site dispute. Contact us to review your site's standing and resolve the dispute."
-				  )
-				: translate( "Contact us to review your site's standing and resolve the dispute." ),
+			),
 			status: 'is-error',
 			contactUrl: localizeUrl( 'https://en.support.wordpress.com/suspended-blogs/' ),
 		},
 		NO_SSL_CERTIFICATE: {
-			message: hasLocalizedText(
+			message: translate(
 				'Certificate installation in progress. Hold tight! We are setting up a digital certificate to allow secure browsing on your site using "HTTPS".'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Hold tight! We are setting up a digital certificate to allow secure browsing on your site, using "HTTPS". Please try again in a few minutes.' )` *ES Score: 7*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/eligibility-warning-rm-conditional-strings)